src/test/java/com/openlattice/rehearsal/organization/AssemblerTest.kt

```diff
@@ -626,6 +665,150 @@ class AssemblerTest : MultipleAuthenticatedUsersBase() {
         loginAs("admin")
     }
 
+    @Test
```

Comment:
Might be worth placing a time limit on this test or breaking out the longer running tasks into separate `@Test` blocks so each build doesn't always take 3min+

Reply:
(assuming we can run tests in parallel)

Reply:
Right now, we are not running rehearsal tests on build. Bamboo can run jobs in parallel, so we could define test groups or something. @geekbeast Can you give me access to Integration tests job pls?

Reply:
@katherinebernstein or @UnsungHero97  should be able to do it. Let's discuss what the plan is here, because if we're going to do something we should make it so rehearsal results are correctly reported (more important than timing atm).